pkgs/build-support/cc-wrapper/cc-wrapper.sh

```diff
@@ -54,6 +54,9 @@ while (( "$n" < "$nParams" )); do
                 c++*) isCxx=1 ;;
             esac
             ;;
+        -target | --target=*)
```

Comment:
Would it not make more sense to do this in the Clang wrapper, where we already have handling for -target?

Reply:
Doh. Probably.